src/sensors/icm20948sensor.cpp

```diff
@@ -210,7 +210,32 @@ void ICM20948Sensor::motionSetup() {
     }
 
     // Might need to set up other DMP functions later, just Quad6/Quad9 for now
-
+    #if MAX_ODR
+    if (USE_6_AXIS)
```

Comment:
While you're here, might as well also rewrite those `if`s into preprocessor `#if`s